### PR TITLE
Update pdf.py PageObject.extractText()

### PR DESCRIPTION
**Pull request by jusdino**
These changes allow for an optional text separator for TJ and Tj operators.

These source alterations were originally suggested in StackOverflow at:
http://stackoverflow.com/questions/11017379/pypdf-ignores-newlines-in-pdf-file
by DSM

I'm just passing along the good suggestion in hopes that the change may become standard in some future version.

### CODE REVIEW

**Comment by MartinThoma** [comment]
Do you have an example where something else than a single whitespace would be desired?

**Comment by MartinThoma** [comment]
By the way: Sorry that it took so long to react! I do realize that you propably don't even remember this PR.

Also: Don't worry about the failing tests; that is expected for this PR.

**Comment by jusdino** [comment]
Yeah, this was a while ago... Ok, I resurrected the project I was working on.

So I was trying to extract text from some form-formatted pdf pages which had newlines separating the text I was interested in, so I used `page.extractText(Tj_sep='\n')` to get it organized the way I needed.

File: PyPDF2/pdf.py [hunk 1]
```diff
@@ -2648,7 +2648,7 @@ def compressContentStreams(self):
                 content = ContentStream(content, self.pdf)
             self[NameObject("/Contents")] = content.flateEncode()
 
-    def extractText(self):
+    def extractText(self, Tj_sep="", TJ_sep=""):
```
Comment (MartinThoma) [inline PyPDF2/pdf.py:2651]:
```suggestion
    def extractText(self, Tj_sep="", TJ_sep=" "):
```

**Pull request merged**